baus/subsidies.py

```diff
@@ -904,15 +904,21 @@ def run_subsidized_developer(feasibility, parcels, buildings, households,
                 buildings.local.loc[index, "deed_restricted_units"] =\
                     int(round(subsidized_units))
 
+                buildings.local.loc[index, "subsidized_units"] = \
+                    buildings.local.loc[index, "deed_restricted_units"] - \
+                    buildings.local.loc[index, "inclusionary_units"]
+
                 # also correct the debug output
-                new_buildings.loc[index, "deed_restricted_units"] =\
+                new_buildings.loc[index, "deed_restricted_units"] = \
                     int(round(subsidized_units))
+                new_buildings.loc[index, "subsidized_units"] = \
+                    new_buildings.loc[index, "deed_restricted_units"] - \
+                    new_buildings.loc[index, "inclusionary_units"]
 
             metadata['deed_restricted_units'] = \
                 new_buildings.loc[index, 'deed_restricted_units']
```

Comment:
@yuqiww does line 912/913 mean that DR = subsidized only here?
(sorry, these comments have been a mess as I'm getting used to the "review" comments)